### DISCUSSION TITLE
Why can't i use fs.configure to set a locationPrefix with collection name?

### DISCUSSION THREAD

**Discussion by wang-0201** [seaweedfs/seaweedfs#4489] [Q&A] [answered]
I want to set save old files in hdd and use ec encode .
So i make a  '/HDD_Storage' dir  and use
```
fs.configure -locationPrefix=/HDD_Storage -collection=HDD_SAVE -disk=hdd -volumeGrowthCount=8 -apply
```
Then i can see this in 'fs.configure'.
But if i use 'collection.list',i can't see any collection.
Now I use seaweedfs "version 8000GB 3.49  linux amd64" without s3 buckets and use redis as filer database.
Maybe we can add -diskType for encode?

**Comment by washcycle** [comment, marked as answer]
The collections don't get created until you upload at least one file to them. @chrislusf I think for POSIX reasons it would make sense to force collection creation when doing this type of `fs.configure`. Reason being that the POSIX location is immediately mountable with `weed mount`. Which if i'm not mistaken will fail if you setup `fs.configure` without uploading a file to that specific collection first.

**Reply by wang-0201** [reply]
Thank you.I had set up successfully .
